server/index.js

```diff
@@ -19,8 +19,21 @@ app.get('/api/health-check', (req, res, next) => {
     .catch(err => next(err));
 });
 
-app.use('/api', (req, res, next) => {
-  next(new ClientError(`cannot ${req.method} ${req.originalUrl}`, 404));
```

Comment:
You didn't need to delete this middleware. It's there for JSON requests to endpoints that don't exist.

Reply:
Oh ok ill add it back in